docs/service-brokers/docs/012-configuration-helm-broker-bundles.md

```diff
@@ -47,6 +47,9 @@ The `meta.yaml` file is mandatory as it contains information about the bundle. S
 |   **documentationURL**  |   false  |                                  The **documentationURL** field described in the [Open Service Broker API][service-metadata].                                    |
 |      **supportURL**     |   false  |                                     The **supportURL** field described in the [Open Service Broker API][service-metadata].                                       |
 |       **imageURL**      |   false  |     The **imageURL** field described in the [Open Service Broker API][service-metadata]. You must provide the image as an SVG.          |
+| **bindingRetrievable**  |   false  |     The **bindingRetrievable** field specifies whether fetching a ServiceBinding using a `GET` call on the resource's endpoint is supported for all plans. The default value is `false`.         |
```

Comment:
please point me to docs that default value is false.

Reply:
I can't find a doc with this. But if we don't specify `bindingRetrievable` in `meta.yaml`, the default value for it is false in the ServiceClass.

Reply:
Service-catalog does not contain any logic with the default value, so it is `false`